### PR TITLE
Added support for coordinate systems: old GSM as Static, real GSM, SM, GSE and time dependent GSM and SM

### PR DESCRIPTION
**Pull request by ElonOlsson**
Making an initial pull request to get it rolling, though the transforms_GSM_SM_TimeDependent.asset 4 files needs to be added to server first if we're allowed. I left it as is for now.

Appreciate feedback on naming of transform assets and the meta tag in the assets. Please try it out 
Screenshot (80)

### CODE REVIEW

File: data/assets/examples/debugcoordinateaxes.asset [hunk 1]
```diff
@@ -1,11 +1,106 @@
...
+      Type = "StaticScale",
+      Scale = EarthRadius * 3.5
+    }
+  },
+  Renderable = {
+    Type = "RenderableCartesianAxes"
+  },
+  GUI = {
+    Name = "GSM Axes",
+    Path = "/Other/Coordinate Systems"
```
Comment (ylvaselling) [inline data/assets/examples/debugcoordinateaxes.asset:46]:
```suggestion
    Path = "/Examples/Coordinate Systems"
```

File: data/assets/examples/debugcoordinateaxes.asset [hunk 2]
```diff
@@ -1,11 +1,106 @@
...
+      Type = "StaticScale",
+      Scale = EarthRadius * 5.5
+    }
+  },
+  Renderable = {
+    Type = "RenderableCartesianAxes"
+  },
+  GUI = {
+    Name = "SM-Time Axes",
+    Path = "/Other/Coordinate Systems"
```
Comment (ylvaselling) [inline data/assets/examples/debugcoordinateaxes.asset:82]:
```suggestion
    Path = "/Examples/Coordinate Systems"
```

File: data/assets/examples/debugcoordinateaxes.asset [hunk 3]
```diff
@@ -1,11 +1,106 @@
...
+      Type = "StaticScale",
+      Scale = EarthRadius * 5.5
+    }
+  },
+  Renderable = {
+    Type = "RenderableCartesianAxes"
+  },
+  GUI = {
+    Name = "GSM-Time Axes",
+    Path = "/Other/Coordinate Systems"
```
Comment (ylvaselling) [inline data/assets/examples/debugcoordinateaxes.asset:100]:
```suggestion
    Path = "/Examples/Coordinate Systems"
```

**Comment by ylvaselling** [comment]
Tried the debugcoordinatesaxes.asset and got these errors
image

Tried the magnetosphere asset and got these errors
image

**Comment by alexanderbock** [comment]
Did you intend it to be in the examples folder?  I thought that we'd move them somewhere into the scene/solarsystem/planets/earth?

**Comment by ElonOlsson** [comment]
For the transforms assets, I moved them to be under planets/earth instead of in planets/earth/magnetosphere like you suggested Alex.
I didn't move the debugcoordinateaxes asset and just added the sgn in the same manner as the ones already in there. We can absolutely move the GUI path to Examples instead of Other.
Your error with the GSM and also assuming SM with time in the name you need the kernels, which I assume we want to put on the server first. The other error, Ylva, seems like you just don't have the class built with cmake?

File: data/assets/examples/debugcoordinateaxes.asset [hunk 4]
```diff
@@ -1,11 +1,107 @@
...
 
 local SunRadius = 6.957E8
 local EarthRadius = 6.371E6
 
+
+local GSEAxes = {
+  Identifier = "GSECoordinateAxes",
+  Parent = GSE_Transforms.GSEReferenceFrame.Identifier,
+  Transform = {
+    Scale ={
```
Comment (alexanderbock) [inline data/assets/examples/debugcoordinateaxes.asset:19]:
```suggestion
    Scale = {
```

File: data/assets/examples/debugcoordinateaxes.asset [hunk 5]
```diff
@@ -1,11 +1,107 @@
...
+    Name = "GSE Axes",
+    Path = "/Other/Coordinate Systems"
+  }
+}
+
+local GSMAxes = {
+  Identifier = "GSMCoordinateAxes",
+  Parent = GSM_SM_Transforms.GSMReferenceFrame.Identifier,
+  Transform = {
+    Scale ={
```
Comment (alexanderbock) [inline data/assets/examples/debugcoordinateaxes.asset:37]:
```suggestion
    Scale = {
```

File: data/assets/examples/debugcoordinateaxes.asset [hunk 6]
```diff
@@ -1,11 +1,106 @@
 local earthTransforms = asset.require("scene/solarsystem/planets/earth/transforms")
```
[the remark this answers is not in the document]
Comment (alexanderbock) [reply data/assets/examples/debugcoordinateaxes.asset:1]: I'm not sure if we need this as an example file in this location at all tbh?  It's not really serving as an example for debugaxes, but for the coordinate system.  And there there 'main' asset file in the scene subfolder would do the same thing in my view

File: data/assets/examples/debugcoordinateaxes.asset [hunk 7]
```diff
@@ -1,11 +1,107 @@
...
+    Name = "GSM Axes",
+    Path = "/Other/Coordinate Systems"
+  }
+}
+
+local SMAxes = {
+  Identifier = "SMCoordinateAxes",
+  Parent = GSM_SM_Transforms.SMReferenceFrame.Identifier,
+  Transform = {
+    Scale ={
```
Comment (alexanderbock) [inline data/assets/examples/debugcoordinateaxes.asset:55]:
```suggestion
    Scale = {
```

File: data/assets/examples/debugcoordinateaxes.asset [hunk 8]
```diff
@@ -1,11 +1,107 @@
...
+    Name = "SM Axes",
+    Path = "/Other/Coordinate Systems"
+  }
+}
+
+local SMTimeAxes = {
+  Identifier = "SMTimeCoordinateAxes",
+  Parent = GSM_SM_Time_Transforms.SMTimeReferenceFrame.Identifier,
+  Transform = {
+    Scale ={
```
Comment (alexanderbock) [inline data/assets/examples/debugcoordinateaxes.asset:73]:
```suggestion
    Scale = {
```

File: data/assets/examples/debugcoordinateaxes.asset [hunk 9]
```diff
@@ -1,11 +1,107 @@
...
+    Name = "SM-Time Axes",
+    Path = "/Other/Coordinate Systems"
+  }
+}
+
+local GSMTimeAxes = {
+  Identifier = "GSMTimeCoordinateAxes",
+  Parent = GSM_SM_Time_Transforms.GSMTimeReferenceFrame.Identifier,
+  Transform = {
+    Scale ={
```
Comment (alexanderbock) [inline data/assets/examples/debugcoordinateaxes.asset:91]:
```suggestion
    Scale = {
```

File: data/assets/scene/solarsystem/planets/earth/kernels/ECLIPDATE.tf [hunk 10]
```diff
@@ -0,0 +1,56 @@
+Earth Mean Ecliptic and Equinox of Date frame (ECLIPDATE)
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/kernels/ECLIPDATE.tf:1]: We'll want to move all of these files into an `identifier` based `asset.resource` as well

File: data/assets/scene/solarsystem/planets/earth/magnetosphere/magnetosphere.asset [hunk 11]
```diff
@@ -28,8 +28,8 @@ local EarthMagnetosphere = {
     ColorMethod = "By Quantity",
     ColorQuantity = 4, -- speed
     ColorTableRanges = { { 50, 300 } },
-    ColorTablePaths = { transferFunction },
     LoadAtRuntime = true,
+    ColorTablePaths = {transferFunction},
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/magnetosphere/magnetosphere.asset:32]:
```suggestion
    ColorTablePaths = { transferFunction },
```

File: data/assets/scene/solarsystem/planets/earth/transforms_GSE.asset [hunk 12]
```diff
@@ -0,0 +1,51 @@
+local transforms = asset.require("scene/solarsystem/planets/earth/transforms")
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/transforms_GSE.asset:1]: The filename should be `transforms_gse.asset` as we only do lowercase filenames

File: data/assets/scene/solarsystem/planets/earth/transforms_GSE.asset [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+local coreKernels = asset.require("spice/core")
+local Eclipdate = asset.resource("kernels/ECLIPDATE.tf")
+local GSEKernel = asset.resource("kernels/GSE.tf")
+
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/transforms_GSE.asset:5]:
```suggestion


```

File: data/assets/scene/solarsystem/planets/earth/transforms_GSE.asset [hunk 14]
```diff
@@ -0,0 +1,51 @@
...
+local Frame = {
+  GSE = "GSE"
+}
+local GSEReferenceFrame = {
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/transforms_GSE.asset:9]:
```suggestion

local GeocentricSolarEcliptic = {
```

File: data/assets/scene/solarsystem/planets/earth/transforms_GSE.asset [hunk 15]
```diff
@@ -0,0 +1,51 @@
...
+  Identifier = "GSEReferenceFrame",
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/transforms_GSE.asset:10]:
```suggestion
  Identifier = "GeocentricSolarEcliptic",
```

No need to say ReferenceFrame as it doesn't make a different to a user whether it is just a reference frame or if it has a renderable attached to it

File: data/assets/scene/solarsystem/planets/earth/transforms_GSM_SM.asset [hunk 16]
```diff
@@ -0,0 +1,74 @@
+local transforms = asset.require("scene/solarsystem/planets/earth/transforms")
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/transforms_GSM_SM.asset:1]: Filename -> `transforms_gsm_sm.asset`

File: data/assets/scene/solarsystem/planets/earth/transforms_GSM_SM.asset [hunk 17]
```diff
@@ -0,0 +1,74 @@
...
+local coreKernels = asset.require("spice/core")
+
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/transforms_GSM_SM.asset:3]:
```suggestion



```

File: data/assets/scene/solarsystem/planets/earth/transforms_GSM_SM.asset [hunk 18]
```diff
@@ -0,0 +1,74 @@
...
+local GSM_Kernel = asset.resource("kernels/GSM.tf")
+local SM_Kernel = asset.resource("kernels/SM.tf")
+
+local Frame = {
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/transforms_GSM_SM.asset:7]:
```suggestion

local Frame = {
```

File: data/assets/scene/solarsystem/planets/earth/transforms_GSM_SM.asset [hunk 19]
```diff
@@ -0,0 +1,74 @@
...
+  GSM = "EARTH_GSM_IGRF2000", -- Geocentric Solar Magnetospheric
+  SM = "EARTH_SM_IGRF2000",
+}
+
+local GSMReferenceFrame = {
+  Identifier = "GSMReferenceFrame",
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/transforms_GSM_SM.asset:13]:
```suggestion
  Identifier = "GeocentricSolarMagnetospheric",
```

File: data/assets/scene/solarsystem/planets/earth/transforms_GSM_static.asset [hunk 20]
```diff
@@ -0,0 +1,52 @@
...
+  Parent = transforms.EarthCenter.Identifier,
+  Transform = {
+    Rotation = {
+      Type = "SpiceRotation",
+      SourceFrame = Frame.GSM_static,
+      DestinationFrame = coreKernels.Frame.Galactic
+    }
+  },
+  GUI = {
+    Name = "Static GSM Reference Frame",
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/transforms_GSM_static.asset:22]:
```suggestion
    Name = "Geocentric Solar Magnetospheric Static",
```

File: data/assets/scene/solarsystem/planets/earth/transforms_GSM_static.asset [hunk 21]
```diff
@@ -0,0 +1,52 @@
...
+    Path = "/Solar System/Planets/Earth",
+    Hidden = true
+  }
+}
+
+
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/transforms_GSM_static.asset:28]:
```suggestion
```

File: data/assets/scene/solarsystem/planets/earth/transforms_GSM_static.asset [hunk 22]
```diff
@@ -0,0 +1,52 @@
...
+
+asset.onInitialize(function()
+  openspace.spice.loadKernel(GSM_Static_Kernel)
+  openspace.addSceneGraphNode(GSMStaticReferenceFrame)
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/transforms_GSM_static.asset:32]:
```suggestion
  openspace.addSceneGraphNode(GeocentricSolarMagnetosphericStatic)
```

File: data/assets/scene/solarsystem/planets/earth/transforms_GSM_static.asset [hunk 23]
```diff
@@ -0,0 +1,52 @@
...
+end)
+
+asset.onDeinitialize(function()
+  openspace.removeSceneGraphNode(GSMStaticReferenceFrame)
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/transforms_GSM_static.asset:36]:
```suggestion
  openspace.removeSceneGraphNode(GeocentricSolarMagnetosphericStatic)
```

File: data/assets/scene/solarsystem/planets/earth/transforms_GSM_static.asset [hunk 24]
```diff
@@ -0,0 +1,52 @@
...
+  openspace.spice.unloadKernel(GSM_Static_Kernel)
+end)
+
+asset.export("Frame", Frame)
+asset.export(GSMStaticReferenceFrame)
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/transforms_GSM_static.asset:41]:
```suggestion
asset.export(GeocentricSolarMagnetosphericStatic)
```

File: data/assets/scene/solarsystem/planets/earth/transforms_GSM_static.asset [hunk 25]
```diff
@@ -0,0 +1,52 @@
...
+
+
+
+asset.meta = {
+Name = "Earth static GSM transforms",
+Version = "1.0",
+Description = "A static version of Earth transforms: GSMReferenceFrame. Geocentric Solar Magnetospheric",
+Author = "CCMC",
+URL = "http://openspaceproject.com",
+License = "MIT license"
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/transforms_GSM_static.asset:51]:
```suggestion
  Name = "Earth static GSM transforms",
  Version = "1.0",
  Description = "A static version of Earth transforms: GSMReferenceFrame. Geocentric Solar Magnetospheric",
  Author = "CCMC",
  URL = "http://openspaceproject.com",
  License = "MIT license"
```

File: data/assets/scene/solarsystem/sun/transforms_heliosphere.asset [hunk 26]
```diff
@@ -27,26 +28,71 @@ local HEEQ180ReferenceFrame = {
   }
 }
 
+local HEEQReferenceFrame = {
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/sun/transforms_heliosphere.asset:31]:
```suggestion
local HeliocentricEarthEquatorial = {
```

File: data/assets/scene/solarsystem/sun/transforms_heliosphere.asset [hunk 27]
```diff
@@ -27,26 +28,71 @@ local HEEQ180ReferenceFrame = {
...
+  Identifier = "HEEQReferenceFrame",
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/sun/transforms_heliosphere.asset:32]:
```suggestion
  Identifier = "HeliocentricEarthEquatorial",
```

File: data/assets/scene/solarsystem/sun/transforms_heliosphere.asset [hunk 28]
```diff
@@ -27,26 +28,71 @@ local HEEQ180ReferenceFrame = {
...
+  Parent = sunTransforms.SunCenter.Identifier,
+  Transform = {
+    Rotation = {
+      Type = "SpiceRotation",
+      SourceFrame = "HEEQ",
+      DestinationFrame = coreKernels.Frame.Galactic
+    }
+  },
+  GUI = {
+    Name = "HEEQ Reference Frame",
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/sun/transforms_heliosphere.asset:42]:
```suggestion
    Name = "Heliocentric Earth Equatorial",
```

File: data/assets/scene/solarsystem/sun/transforms_heliosphere.asset [hunk 29]
```diff
@@ -27,26 +28,71 @@ local HEEQ180ReferenceFrame = {
...
+    Path = "/Solar System/Sun",
+    Hidden = true
+  }
+}
+
+local HEEReferenceFrame = {
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/sun/transforms_heliosphere.asset:48]:
```suggestion
local HeliocentricEarthEcliptic = {
```

File: data/assets/scene/solarsystem/sun/transforms_heliosphere.asset [hunk 30]
```diff
@@ -27,26 +28,71 @@ local HEEQ180ReferenceFrame = {
...
+  Identifier = "HEEReferenceFrame",
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/sun/transforms_heliosphere.asset:49]:
```suggestion
  Identifier = "HeliocentricEarthEcliptic",
```

File: data/assets/scene/solarsystem/sun/transforms_heliosphere.asset [hunk 31]
```diff
@@ -27,26 +28,71 @@ local HEEQ180ReferenceFrame = {
...
+  Parent = sunTransforms.SunCenter.Identifier,
+  Transform = {
+    Rotation = {
+      Type = "SpiceRotation",
+      SourceFrame = "HEE",
+      DestinationFrame = coreKernels.Frame.Galactic
+    }
+  },
+  GUI = {
+    Name = "HEE Reference Frame",
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/sun/transforms_heliosphere.asset:59]:
```suggestion
    Name = "Heliocentric Earth Ecliptic",
```

File: data/assets/scene/solarsystem/sun/transforms_heliosphere.asset [hunk 32]
```diff
@@ -27,26 +28,71 @@ local HEEQ180ReferenceFrame = {
...
 
 asset.onInitialize(function()
-  openspace.spice.loadKernel(kernel)
+  openspace.spice.loadKernel(heeq180Kernel)
+  openspace.spice.loadKernel(heeqKernel)
+  openspace.spice.loadKernel(heeKernel)
 
   openspace.addSceneGraphNode(HEEQ180ReferenceFrame)
+  openspace.addSceneGraphNode(HEEQReferenceFrame)
+  openspace.addSceneGraphNode(HEEReferenceFrame)
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/sun/transforms_heliosphere.asset:74]:
```suggestion
  openspace.addSceneGraphNode(HeliocentricEarthEcliptic)
```

File: data/assets/scene/solarsystem/sun/transforms_heliosphere.asset [hunk 33]
```diff
@@ -27,26 +28,71 @@ local HEEQ180ReferenceFrame = {
...
 end)
 
 asset.onDeinitialize(function()
+  openspace.removeSceneGraphNode(HEEReferenceFrame)
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/sun/transforms_heliosphere.asset:78]:
```suggestion
  openspace.removeSceneGraphNode(HeliocentricEarthEcliptic)
```

File: data/assets/scene/solarsystem/sun/transforms_heliosphere.asset [hunk 34]
```diff
@@ -27,26 +28,71 @@ local HEEQ180ReferenceFrame = {
...
+  openspace.removeSceneGraphNode(HEEQReferenceFrame)
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/sun/transforms_heliosphere.asset:79]:
```suggestion
  openspace.removeSceneGraphNode(HeliocentricEarthEquatorial)
```

File: data/assets/scene/solarsystem/sun/transforms_heliosphere.asset [hunk 35]
```diff
@@ -27,26 +28,71 @@ local HEEQ180ReferenceFrame = {
...
+
 
 asset.onInitialize(function()
-  openspace.spice.loadKernel(kernel)
+  openspace.spice.loadKernel(heeq180Kernel)
+  openspace.spice.loadKernel(heeqKernel)
+  openspace.spice.loadKernel(heeKernel)
 
   openspace.addSceneGraphNode(HEEQ180ReferenceFrame)
+  openspace.addSceneGraphNode(HEEQReferenceFrame)
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/sun/transforms_heliosphere.asset:73]:
```suggestion
  openspace.addSceneGraphNode(HeliocentricEarthEquatorial)
```

File: data/assets/scene/solarsystem/sun/transforms_heliosphere.asset [hunk 36]
```diff
@@ -1,10 +1,11 @@
...
 local coreKernels = asset.require("spice/core")
 
+local heeq180Kernel = asset.resource("kernels/HEEQ180.tf")
+local heeqKernel = asset.resource("kernels/HEEQ.tf")
+local heeKernel = asset.resource("kernels/HEE.tf")
 
 
-local kernel = asset.resource("kernels/HEEQ180.tf")
-
 local HEEQ180ReferenceFrame = {
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/sun/transforms_heliosphere.asset:9]:
```suggestion
local HeliocentricEarthEquatorial180 = {
```

File: data/assets/scene/solarsystem/sun/transforms_heliosphere.asset [hunk 37]
```diff
@@ -1,10 +1,11 @@
...
   Identifier = "HEEQ180ReferenceFrame",
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/sun/transforms_heliosphere.asset:10]:
```suggestion
  Identifier = "HeliocentricEarthEquatorial180",
```

Comment (alexanderbock) [reply data/assets/scene/solarsystem/sun/transforms_heliosphere.asset:10]: Can't mark it in GitHub but the GUI.Name should be "Heliocentric Earth Equatorial"

File: data/assets/scene/solarsystem/sun/transforms_heliosphere.asset [hunk 38]
```diff
@@ -27,26 +28,71 @@ local HEEQ180ReferenceFrame = {
...
 
   openspace.addSceneGraphNode(HEEQ180ReferenceFrame)
+  openspace.addSceneGraphNode(HEEQReferenceFrame)
+  openspace.addSceneGraphNode(HEEReferenceFrame)
 end)
 
 asset.onDeinitialize(function()
+  openspace.removeSceneGraphNode(HEEReferenceFrame)
+  openspace.removeSceneGraphNode(HEEQReferenceFrame)
   openspace.removeSceneGraphNode(HEEQ180ReferenceFrame)
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/sun/transforms_heliosphere.asset:80]:
```suggestion
  openspace.removeSceneGraphNode(HeliocentricEarthEquatorial180)
```

Comment (alexanderbock) [inline data/assets/scene/solarsystem/sun/transforms_heliosphere.asset:80]:
```suggestion
  openspace.removeSceneGraphNode(HeliocentricEarthEquatorial180)
```

**Review by alexanderbock — COMMENTED** [review]
All of the new coordinate systems desparately needs a GUI.Description that describes what they do to a semi-lay audience. You can probably take the beginning of the frame kernel definitions and put that into the description.

Also all of the frame kernels should be uploaded to the server and then used with an `asset.resource` instead

**Review by alexanderbock — COMMENTED** [review]
All of the new coordinate systems desparately needs a GUI.Description that describes what they do to a semi-lay audience. You can probably take the beginning of the frame kernel definitions and put that into the description.

Also all of the frame kernels should be uploaded to the server and then used with an `asset.resource` instead

File: data/assets/scene/solarsystem/planets/earth/transforms_gse.asset [hunk 39]
```diff
@@ -1,10 +1,15 @@
 local transforms = asset.require("scene/solarsystem/planets/earth/transforms")
 local coreKernels = asset.require("spice/core")
-local Eclipdate = asset.resource("kernels/ECLIPDATE.tf")
-local GSEKernel = asset.resource("kernels/GSE.tf")
 
 
 
+local earthcentricKernels = asset.resource({
+  Name = "Earthcenteric frame kernels",
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/transforms_gse.asset:7]:
```suggestion
  Name = "Earthcentric Frame Kernels",
```

File: data/assets/scene/solarsystem/planets/earth/transforms_gsm_sm.asset [hunk 40]
```diff
@@ -1,10 +1,14 @@
 local transforms = asset.require("scene/solarsystem/planets/earth/transforms")
 local coreKernels = asset.require("spice/core")
 
-local GSM_Kernel = asset.resource("kernels/GSM.tf")
-local SM_Kernel = asset.resource("kernels/SM.tf")
 
 
+local earthcentricKernels = asset.resource({
+  Name = "Earthcenteric frame kernels",
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/transforms_gsm_sm.asset:7]:
```suggestion
  Name = "Earthcentric Frame Kernels",
```

File: data/assets/scene/solarsystem/planets/earth/transforms_gsm_sm_timedependent.asset [hunk 41]
```diff
@@ -4,8 +4,6 @@ local coreKernels = asset.require("spice/core")
 local GSM_Kernel = asset.resource("kernels/GSM_time.tf")
 local SM_Kernel = asset.resource("kernels/SM_time.tf")
 
-
-
 local kernels = {
   "C:/Users/eolsson/Downloads/earthnpole_19500101_20251231_v01.bsp",
   "C:/Users/eolsson/Downloads/earthnpole_runout.bsp",
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/transforms_gsm_sm_timedependent.asset:9]: These entries (and other like it should go as well)

File: data/assets/scene/solarsystem/planets/earth/transforms_gsm_static.asset [hunk 42]
```diff
@@ -1,9 +1,14 @@
 local transforms = asset.require("scene/solarsystem/planets/earth/transforms")
 local coreKernels = asset.require("spice/core")
 
-local GSM_Static_Kernel = asset.resource("kernels/GSM_static.tf")
 
 
+local earthcentricKernels = asset.resource({
+  Name = "Earthcenteric frame kernels",
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/transforms_gsm_static.asset:7]:
```suggestion
  Name = "Earthcentric Frame Kernels",
```

**Comment by alexanderbock** [comment]
@ElonOlsson

**Pull request merged**